sensecap-controller-android/app/src/main/java/com/github/infamousriddles/sensecapcontroller/network/SenseCAPService.kt

```diff
@@ -0,0 +1,19 @@
+package com.github.infamousriddles.sensecapcontroller.network
+
+import retrofit2.Response
+import retrofit2.http.GET
+
+interface SenseCAPService {
+
+    @GET("/sensecap/shutdown")
```

Comment:
@charbgr `retrofit.baseUrl` already includes `/sensecap`. Are you sure you need the prefix here as well? IMO we should remove it from the `baseUrl` in order to make it more flexible for additional hotspot brands!